recipes/ipopt/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+THIRDPARTY=(Metis Mumps)
```

Comment:
Metis is optional in Ipopt. You'd need to use Mumps 5.x for it to work with Metis 5.x (which has a normal Apache license), and I don't think Ipopt has been updated yet to work with Mumps 5.x. Maybe if you build them all separately it might work as an external library instead of building the vendored copy under ThirdParty, if there aren't API changes. I think homebrew-science might be building Mumps and Ipopt that way.

Probably simplest to leave out Metis for now if you want to redistribute binaries (kind of the point of conda-forge), and revisit with a todo for building latest versions of things separately. If you're going to link against MKL then you should be able to use the MKL implementation of Pardiso for Ipopt anyway, which won't need Metis.